bugsnag-android-core/src/main/java/com/bugsnag/android/DeliveryDelegate.java

```diff
@@ -43,12 +43,32 @@ void deliver(@NonNull Event event) {
         }
 
         if (event.isUnhandled()) {
-            cacheEvent(event, true);
+            // should only send unhandled errors if they don't terminate the process or
+            // if they can't be reported on next launch. ANRs don't terminate the process
+            // directly and stackoverflow can't be persisted on disk
```

Comment:
Can you investigate this limitation and either clarify the comment (as it still persists these errors) or allow stack overflows to be sent as normal?

Reply:
Clarified comment